challenges/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,7 @@
+function arrayReverse(arr) {
+  let reversedArray = [];
+  for(let i = arr.length -1; i >= 0; i --) {
+    reversedArray.push(arr[i]);
```

Comment:
cant use push here. think about where(the index) in the reversedArray you need to put the numbers